modules/examiner/jsx/editExaminer.js

```diff
@@ -0,0 +1,333 @@
+      formData: {},
+      statusOptions: {
+        null: "N/A",
+        notCertified: "Not Certified",
+        inTraining: "In Training",
+        certified: "Certified"
+      }
+    };
+
+    /**
+     * Bind component instance to custom methods
+     */
+    this.fetchData = this.fetchData.bind(this);
+    this.setFormData = this.setFormData.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  componentDidMount() {
+    this.fetchData();
+  }
+
+  /**
+   * Retrieve data from the provided URL and save it in state
+   */
+  fetchData() {
+    const self = this;
+    let formData = {};
+    $.ajax({
+      url: this.props.dataURL,
+      type: 'GET',
+      dataType: 'json',
+      data: {
+        identifier: this.props.examinerID
+      },
+      success: data => {
+        const instruments = data.instruments;
+        for (let instrumentID in instruments) {
+          if (instruments.hasOwnProperty(instrumentID)) {
+            formData[instrumentID] = {
+              status: data.instruments[instrumentID].pass,
+              date: data.instruments[instrumentID].date_cert,
+              comments: data.instruments[instrumentID].comment
+            };
+          }
+        }
+        self.setState({
+          data: data,
+          formData: formData,
+          isLoaded: true
+        });
+      },
+      error: error => {
+        console.error(error);
+      }
+    });
+  }
+
+  /**
+   * Store the value of the element in this.state.formData
+   *
+   * @param {string} formElement  name of the form element
+   * @param {string} value        value of the form element
+   */
+  setFormData(formElement, value) {
+    let formData = this.state.formData;
+    let split = formElement.split("_");
+    let key = split[0];
+    let element = split[1];
+    formData[key][element] = value;
+    this.setState({
+      formData: formData
+    });
+  }
+
+  /**
+   * Handles the submission of the Add Examiner form
+   *
+   * @param {event} e event of the form
+   */
+  handleSubmit(e) {
+    e.preventDefault();
+    let formData = this.state.formData;
+    let formObject = new FormData();
+    for (let testID in formData) {
+      if (formData.hasOwnProperty(testID) &&
+        JSON.stringify(formData[testID]) !== '{}') {
+        let instrumentData = JSON.stringify(formData[testID]);
+        formObject.append(testID, instrumentData);
+      }
+    }
+    formObject.append('identifier', this.props.examinerID);
+    $.ajax({
+      type: 'POST',
+      url: `${loris.BaseURL}/examiner/editCertification`,
+      data: formObject,
+      cache: false,
+      contentType: false,
+      processData: false,
+      success: data => {
+        swal('Success!', 'Changes to certification submitted.', 'success');
+        this.fetchData();
+      },
+      error: error => {
+        console.error(error);
+        let message = error.responseText;
+        swal('Error!', message, 'error');
+      }
+    });
+  }
+
+  render() {
```

Comment:
this render function has a lot in it. i'd suggest getting some of the subcomponents from functions to make it a bit cleaner